tests/test_view_mode.py

```diff
@@ -1,10 +1,17 @@
 import pytest
-from todo_app.board import Board
-from todo_app.data import trello_items
+from todo_app.data import mongo_items
 from todo_app.viewModel import ViewModel
 from todo_app.item import Item
+from todo_app import app
+from dotenv import load_dotenv, find_dotenv
 
-TEST_BOARD = Board ('ID', 'KEY', 'TOKEN', 'TO_DO_LIST_ID' , 'DONE_LIST_ID')
+
+
+def get_test_app():
```

Comment:
Because of the complexity in the ViewModel, this has ended up being basically an integration test - handled differently to our suggestion but equally well!
That said, hopefully the comment below about the alternative way to structure the view model makes sense when considering how much simpler this test could then be (we just directly create a view model using the test_items, and don't need to worry about mocking!)